doc/source/whatsnew/v3.0.0.rst

```diff
@@ -401,6 +401,7 @@ Other API changes
 - Index set operations (like union or intersection) will now ignore the dtype of
   an empty ``RangeIndex`` or empty ``Index`` with object dtype when determining
   the dtype of the resulting Index (:issue:`60797`)
+- :meth:`DataFrame.query` does not accept ``**kwargs`` anymore and requires passing keywords for desired arguments (:issue:`61405`)
```

Comment:
Since this is fully backwards compatible, I think we do not need to mention this in the whatsnew.